### PR TITLE
builstep alias configuration in reactor config map

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-7530

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates

### CODE REVIEW

File: tests/plugins/test_reactor_config.py [hunk 1]
```diff
@@ -54,12 +55,20 @@
 from flexmock import flexmock
 
 
+class X(object):
```
Comment (twaugh) [inline tests/plugins/test_reactor_config.py]: Can we use StubInsideBuilder for this?

Comment (rcerven) [reply tests/plugins/test_reactor_config.py]: sure, updated

Comment (MartinBasti) [reply tests/plugins/test_reactor_config.py]: Could you remove `X` class then?

Comment (rcerven) [reply tests/plugins/test_reactor_config.py]: deleted

**Comment by rcerven** [comment]
rn updated

**Pull request merged**